webapp/src/Controller/Jury/RejudgingController.php

```diff
@@ -133,7 +133,7 @@ public function indexAction(): Response
                 $status = $rejudging->getValid() ? 'applied' : 'canceled';
                 $sort_order = 2;
             } elseif ($todo > 0) {
-                $perc   = (int)(100 * ((double)$done / (double)($done + $todo)));
```

Comment:
Why is this better?

Reply:
They are the same type so php cs fixer picks one. See https://cs.symfony.com/doc/rules/cast_notation/short_scalar_cast.html